src/sentry/integrations/bitbucket/issues.py

```diff
@@ -20,25 +20,23 @@ def get_issue_url(self, key):
         repo, issue_id = key.split('#')
         return u'https://bitbucket.org/{}/issues/{}'.format(repo, issue_id)
 
-    def get_repo_choices(self, **kwargs):
+    def get_persisted_default_config_fields(self):
+        return ['repo']
+
+    def get_repo_choices(self, group):
         try:
             repos = self.get_repositories()
         except ApiError:
             return [], None
 
         repo_choices = [(repo['identifier'], repo['name']) for repo in repos]
-        params = kwargs.get('params', {})
-
-        try:
-            default_repo = params.get('repo', repo_choices[0][0])
```

Comment:
Do we no longer need to pull things out of the parameters?

Reply:
So, I think bb wasn't really using it, but I'll go ahead and add that back in and just replace `repo_choices[0][0]`